server/handlebars/public/script.js

```diff
@@ -0,0 +1,33 @@
+const iframeClass = `support-levels-${document.currentScript.getAttribute(
+    'pattern'
+)}`;
+
+const iframeCode = link =>
+    `<iframe
+        class="${iframeClass}"
+        src="${link}"
+        height="500"
+        allow="clipboard-write"
+        style="border-style: none; width: 100%;">
+    </iframe>`;
```

Comment:
Could we format the string like this? This formatting would require less correction when embedded.

```
    `  <iframe
    class="${iframeClass}"
    src="${link}"
    height="500"
    allow="clipboard-write"
    style="border-style: none; width: 100%;">
  </iframe>`;
```